src/commands/snapshot.ts

```diff
@@ -56,20 +56,21 @@ export default class Snapshot extends PercyCommand {
 
     const {args, flags} = this.parse(Snapshot)
 
-    const isWindows = process.platform === 'win32'
-
     const snapshotDirectory = args.snapshotDirectory as string
     const port = flags.port as number
     const staticServerPort = port + 1
     const networkIdleTimeout = flags['network-idle-timeout'] as number
     const baseUrl = flags['base-url'] as string
-    const ignoreFilesRegex = flags['ignore-files'] as string
-    const snapshotFilesRegex = flags['snapshot-files'] as string
+    const rawIgnoreGlob = flags['ignore-files'] as string
+    const rawSnapshotGlob = flags['snapshot-files'] as string
```

Comment:
these globs will come in as strings and need to be converted to an array of strings to work with the globby package